Octokit/Models/Request/IssueRequest.cs

```diff
@@ -140,6 +140,27 @@ public enum ItemState
         /// </summary>
         All
     }
+    /// <summary>
+    /// Range of States
+    /// </summary>
+    public enum ItemState
+    {
+        /// <summary>
+        /// Issues that are open
+        /// </summary>
+        Open,
+
+        /// <summary>
+        /// Issues that are closed
+        /// </summary>
+        Closed,
+
+        /// <summary>
+        /// All the issues. The option is Obsolete
+        /// </summary>
+        [Obsolete("The method is Obsolete. Dont specify the state in case of all queries")]
```

Comment:
As per my other comment Im thinking we actually can just remove this guy and not need to Obsolete it.  Waiting for @shiftkey to confirm.  If we DO need to obsolete it though, the warning message should be reworded, since this is not a "method".  Something like
> This value is obsolete and will be removed in a future version.  For Request operations please use ItemStateFilter.All.  For Search, Create and Update operations, "all" is not a valid option.